Cannot install in windows 8.1 x64

Cannot compile in windows
Atom version 0.179.0
MS VC++ 2010 Express installed
Python 2.7.9 x64 installled

```
Installing “pdf-view@0.17.0” failed.Hide output…

> runas@2.0.0 install C:\Users\Marco\AppData\Local\Temp\apm-install-dir-115116-4704-bsagyz\node_modules\pdf-view\node_modules\pathwatcher\node_modules\runas
> node-gyp rebuild


C:\Users\Marco\AppData\Local\Temp\apm-install-dir-115116-4704-bsagyz\node_modules\pdf-view\node_modules\pathwatcher\node_modules\runas>node "C:\Users\Marco\AppData\Local\atom\app-0.179.0\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild 
Los proyectos de esta soluci�n se van a compilar de uno en uno. Para habilitar la compilaci�n en paralelo, agregue el modificador "/m".
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V120\Microsoft.CppBuild.targets(396,5): warning MSB8029: The Intermediate directory or Output directory cannot reside under the Temporary directory as it could lead to issues with incremental build. [C:\Users\Marco\AppData\Local\Temp\apm-install-dir-115116-4704-bsagyz\node_modules\pdf-view\node_modules\pathwatcher\node_modules\runas\build\runas.vcxproj]
  main.cc
  runas_win.cc
c:\users\marco\.atom\.node-gyp\.node-gyp\0.21.0\deps\uv\include\uv-win.h(32): fatal error C1083: Cannot open include file: 'winsock2.h': No such file or directory (..\src\main.cc) [C:\Users\Marco\AppData\Local\Temp\apm-install-dir-115116-4704-bsagyz\node_modules\pdf-view\node_modules\pathwatcher\node_modules\runas\build\runas.vcxproj]

gypnpm ERR! Windows_NT 6.2.9200
npm ERR! argv "C:\\Users\\Marco\\AppData\\Local\\atom\\app-0.179.0\\resources\\app\\apm\\bin\\\\node.exe" "C:\\Users\\Marco\\AppData\\Local\\atom\\app-0.179.0\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\Marco\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\Marco\\.atom\\.apmrc" "install" "C:\\Users\\Marco\\AppData\\Local\\Temp\\d-115116-4704-17ggql4\\package.tgz" "--target=0.21.0" "--arch=ia32" "--msvs_version=2013"
npm ERR! node v0.10.35
npm ERR! npm  v2.3.0
npm ERR! code ELIFECYCLE

npm ERR! runas@2.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the runas@2.0.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the runas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls runas
npm ERR! There is likely additional logging output above.
```

I confirm that the plugin is not working on Win 8.1 x64. I get the following error 
```
> runas@2.0.0 install C:\Users\bennehar\AppData\Local\Temp\apm-install-dir-115226-6420-zy422t\node_modules\pdf-view\node_modules\pathwatcher\node_modules\runas
> node-gyp rebuild


C:\Users\bennehar\AppData\Local\Temp\apm-install-dir-115226-6420-zy422t\node_modules\pdf-view\node_modules\pathwatcher\node_modules\runas>node "C:\Users\bennehar\AppData\Local\atom\app-0.188.0\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild 

gypnpm ERR! Windows_NT 6.2.9200
npm ERR! argv "C:\\Users\\bennehar\\AppData\\Local\\atom\\app-0.188.0\\resources\\app\\apm\\bin\\\\node.exe" "C:\\Users\\bennehar\\AppData\\Local\\atom\\app-0.188.0\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\bennehar\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\bennehar\\.atom\\.apmrc" "install" "C:\\Users\\bennehar\\AppData\\Local\\Temp\\d-115226-6420-zg9ix8\\package.tgz" "--target=0.21.0" "--arch=ia32" "--msvs_version=2013"
npm ERR! node v0.10.35
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! runas@2.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the runas@2.0.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the runas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls runas
npm ERR! There is likely additional logging output above.
```

--- a/lib/pdf-editor-view.coffee
+++ b/lib/pdf-editor-view.coffee
@@ -2,9 +2,8 @@
 fs = require 'fs-plus'
 path = require 'path'
 require './../node_modules/pdf.js/build/generic/build/pdf.js'
-{File} = require 'pathwatcher'
 _ = require 'underscore-plus'
-{Disposable, CompositeDisposable} = require 'atom'
+{File, Disposable, CompositeDisposable} = require 'atom'
 
 PDFJS.workerSrc = "file://" + path.resolve(__dirname, "../node_modules/pdf.js/build/generic/build/pdf.worker.js")
 

--- a/package.json
+++ b/package.json
@@ -13,7 +13,6 @@
   },
   "dependencies": {
     "fs-plus": "2.x",
-    "pathwatcher": "^4.0.1",
     "pdf.js": "git+https://github.com/mozilla/pdf.js.git#v1.0.907",
     "underscore-plus": "^1.6",
     "atom-space-pen-views": "^2.0.3"

